packages/cli/tests/cli/validation/source-without-address-is-valid/generated/schema.ts

```diff
@@ -0,0 +1,44 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Same as `ExampleContract.ts`, do you know why is this being added?

Reply:
I forget to update the `.gitignore` paths